current_pos: [5.544445, 5.544445, 0.0]
current_vel: [-1.0, +1.0, 0.0]
target: [2.0, 7.54, 0.0]

behavior: 3
# FLEE 0
# SEEK 1
# STOP 2
# ARRIVAL 3